app/controllers/namespaces_controller.rb

```diff
@@ -28,6 +28,11 @@ def show
 
     authorize @namespace
     @repositories = @namespace.repositories.page(params[:page])
+    @namespace_entity = API::Entities::Namespaces.represent(
```

Comment:
I think that it would be better to move this entirely to the grape API, and then simply remove the code from this controller in this regard.

Reply:
Are we gonna start moving the non GET actions to the API?

Reply:
I think it would be a good place to start :-) I'll start with it asap

Reply:
I'll try then. It's good for me to practice and learn something that I'm not too familiar with. Don't worry! :)